go/teams/rpc_exim.go

```diff
@@ -11,36 +11,34 @@ import (
 	keybase1 "github.com/keybase/client/go/protocol/keybase1"
 )
 
-func (t *Team) ExportToTeamPlusApplicationKeys(ctx context.Context, idTime keybase1.Time, application keybase1.TeamApplication) (teamPlusApplicationKeys keybase1.TeamPlusApplicationKeys, err error) {
+func (t *Team) ExportToTeamPlusApplicationKeys(ctx context.Context, idTime keybase1.Time, application keybase1.TeamApplication) (ret keybase1.TeamPlusApplicationKeys, err error) {
 	applicationKeys, err := t.AllApplicationKeys(ctx, application)
 	if err != nil {
-		return
+		return ret, err
```

Comment:
Sorry I think I put that in a confusing way. I was suggesting we remove the named returns from the signature of the function. But I definitely don't care very much :)